test/python_bindings.py

```diff
@@ -0,0 +1,435 @@
+import math
+import unittest
+import singularity_eos
+import numpy as np
+from numpy.testing import assert_allclose
+
+class EOSTestBase(object):
+    def assertIsClose(self, a, b, eps=5e-2):
+        rel = abs(b - a) / (abs(a + b) + 1e-20)
```

Comment:
This is fine for now, but we may want to convince ourselves at some later date that 1e-20 is the right number here.

Reply:
I originally tried to use some standard comparison functions, but due to failures then reverted to the same what the C++ tests were doing.